src/api/controllers/stockController.js

```diff
@@ -20,4 +20,15 @@ export default class StockController {
         }
 
     }
+
+    async postStockAnalysisData(stockData, response) {
```

Comment:
remember we changed from analysis to candlestick

Reply:
Done